.changeset/config.json

```diff
@@ -10,7 +10,7 @@
   "ignore": [
     "sui-explorer",
     "sui-wallet",
-    "sui-wallet-adapter",
+    "sui-wallet-adapter-demo",
```

Comment:
why demo?

Reply:
This is the demo app for the wallet adapter, so we're marking to ignore it from versioning. Although future versions of changesets will mean we don't need to spell it out.